src/components/views/settings/tabs/room/RolesRoomSettingsTab.tsx

```diff
@@ -252,6 +257,11 @@ export default class RolesRoomSettingsTab extends React.Component<IProps> {
         if (SettingsStore.getValue("feature_pinning")) {
             plEventsToLabels[EventType.RoomPinnedEvents] = _td("Manage pinned events");
         }
+        // MSC33401: Native Group VoIP signalling
+        if (SettingsStore.getValue("feature_group_calls")) {
+            plEventsToLabels[ElementCall.CALL_EVENT_TYPE.name] = _td("Start Element calls");
+            plEventsToLabels[ElementCall.MEMBER_EVENT_TYPE.name] = _td("Join Element calls");
```

Comment:
These should be using the `brand` variable

Reply:
See https://github.com/matrix-org/matrix-react-sdk/pull/9333#issuecomment-1268864360 - the product decision was that 'Element Call' should be an immutable brand

Reply:
Hardcoding doesn't fulfill our other obligations, though. Much like how "Element" is the product name, we should be able to swap "Element Call" out for a brand variable as well. It doesn't need to be fully advertised, however.